src/components/NewClusterModal.js

```diff
@@ -115,7 +118,7 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
     )
     this.state = {
       profile: matchingProfile ? matchingProfile.name : 'custom',
-      jupyterUserScriptUri: '', customEnvImage: '', viewMode: undefined, isDeleteView: false,
+      jupyterUserScriptUri: '', customEnvImage: '', projSpecImage: '', viewMode: undefined, isDeleteView: false,
```

Comment:
I struggled with this decision and went back and forth about just using the customEnvImage. I ultimately went with two pieces of state so that the custom and project-specific environments would be more decoupled as it seemed to be emphasized in the requirement that this BYORstudio thing should be clearly separated from existing notebook stuff, but I am certainly open to changing it

Reply:
Upon further reflection and discussion with Isaac I will remove the extra state and just use customEnvImage. Thanks for the call out!